code-buddy.html

```diff
@@ -0,0 +1,161 @@
+              alt="Plus"
+              class="code-buddy__plus"
+            />
+          </li>
+          <li class="teams__items">Mandy Schwarze</li>
+        </ul>
+      </section>
+      <section class="teams">
+        <ul class="teams__list">
+          <li class="teams__items code-buddy">
+            Victoria Wika<img
+              src="./icons/Plus.svg"
+              alt="Plus"
+              class="code-buddy__plus"
+            />
+          </li>
+          <li class="teams__items">Eileen</li>
+        </ul>
+      </section>
+      <section class="teams">
+        <ul class="teams__list">
+          <li class="teams__items code-buddy">
+            Marcus Hohlbein<img
+              src="./icons/Plus.svg"
+              alt="Plus"
+              class="code-buddy__plus"
+            />
+          </li>
+          <li class="teams__items">Melissa Gries</li>
+        </ul>
+      </section>
+      <section class="teams">
+        <ul class="teams__list">
+          <li class="teams__items code-buddy">
+            Paula Rösler<img
+              src="./icons/Plus.svg"
+              alt="Plus"
+              class="code-buddy__plus"
+            />
+          </li>
+          <li class="teams__items">Anastasiya Vasileuskaya</li>
+        </ul>
+      </section>
+      <section class="teams">
+        <ul class="teams__list">
+          <li class="teams__items code-buddy">
+            Daniel Wiesner<img
+              src="./icons/Plus.svg"
+              alt="Plus"
+              class="code-buddy__plus"
+            />
+          </li>
+          <li class="teams__items">Nils Ammersken</li>
+        </ul>
+      </section>
+      <section class="teams">
+        <ul class="teams__list">
+          <li class="teams__items code-buddy">
+            Robin Steinhauser<img
+              src="./icons/Plus.svg"
+              alt="Plus"
+              class="code-buddy__plus"
+            />
+          </li>
+          <li class="teams__items">Anja Seeba</li>
+        </ul>
+      </section>
+      <section class="teams">
+        <ul class="teams__list">
+          <li class="teams__items code-buddy">
+            Anneke Schütte<img
+              src="./icons/Plus.svg"
+              alt="Plus"
+              class="code-buddy__plus"
+            />
+          </li>
+          <li class="teams__items">Berenice Pendzialek</li>
+        </ul>
+      </section>
+      <section class="teams">
+        <ul class="teams__list">
+          <li class="teams__items code-buddy">
+            Laura Strehlow<img
+              src="./icons/Plus.svg"
+              alt="Plus"
+              class="code-buddy__plus"
+            />
+          </li>
+          <li class="teams__items">Jerry Erbs/Lene Frei</li>
+        </ul>
+      </section>
+    </main>
+    <nav class="navigation">
+      <a href="/index.html"
+        ><img
+          src="./icons/DashboardIcon.svg"
+          alt="Dashboard"
+          class="navigation__icon-inactive"
```

Comment:
The correct usage of BEM would be:
```suggestion
          class="navigation__icon--inactive"
```